src/exception/fail.cpp

```diff
@@ -0,0 +1,40 @@
+//
+// Created by Dawid Drozd aka Gelldur on 6/30/16.
+//
+
+#include "fail.hpp"
+
+#include <stdexcept>
+
+namespace commoncpp::exception::fail
+{
+
+std::string trimFromSrcFolder(std::string_view where)
+{
+	auto srcPosition = where.rfind("src/");
+	if(srcPosition != std::string::npos)
+	{
+		return std::string{where.substr(srcPosition + 4)};
+	}
+	return std::string{where};
+}
+
+void if_false(bool valueToCheck, std::string_view where)
```

Comment:
Naming of this method and the one below is quite weird, I would expect it to accept a callback and work in a if_false(callback), a better name would be throw_if_false.
Also, it could in theory be templated as
template<bool expected_condition>
void throw_if(const bool valueToCheck, const std::string_view where)
{...}
Which IMO is a bit better design

Reply:
I design it to play more nicely with namespace e.g. `exception::fail::if_false(...)`, but sure thanks for suggestion :) I only don't get the point what you mean by using template ? What for template in such case ?